ex2_services/Services/source/cli/fs_utils.c

```diff
@@ -299,7 +333,7 @@ static BaseType_t prvTRANSACTCommand(char *pcWriteBuffer, size_t xWriteBufferLen
 
 static BaseType_t prvCPCommand(char *pcWriteBuffer, size_t xWriteBufferLen, const char *pcCommandString) {
     BaseType_t fromParameterLen;
-    char *copyFrom = FreeRTOS_CLIGetParameter(
+    char *copyFrom = FreeRTOS_CLIGetParameter( // I know there's a warning here, a null terminator needs to be placed in the array :(
```

Comment:
This whole FreeRTOS CLI thing is pretty obtuse :-/ Did you say you wanted to replace it with a DIY version that was 1/10th the size and 1/10th the complexity?

Reply:
The CLIGetParameter returns a const pointer but I had to cast it out of the const (I know I should be fired for that offense) because the command string stores a space where reliance-edge expects to see a null terminator for the filename